js_modules/dagit/packages/core/codegen.ts

```diff
@@ -0,0 +1,25 @@
+import {CodegenConfig} from '@graphql-codegen/cli';
+
+const config: CodegenConfig = {
+  schema: './src/graphql/schema.graphql',
+  documents: ['src/**/*.tsx'],
+  ignoreNoDocuments: true, // for better experience with the watcher
+  generates: {
+    './src/graphql/': {
+      preset: 'client',
+      config: {
+        nonOptionalTypename: true,
+        namingConvention: {
+          enumValues: 'keep',
```

Comment:
Maintain `ALL_CAPS` enum keys for now. We can remove this in a later refactor step to simplify the codegen config, and update our enums to use `PascalCase`.